### DISCUSSION TITLE
测试的结果都是“-”是怎么回事啊

### DISCUSSION THREAD

**Discussion by claire360** [open-compass/opencompass#1203] [Q&A]
![image](https://github.com/open-compass/opencompass/assets/64513343/007418c6-c3e9-4863-b324-0467cf552e13)

结果如上

给的命令是这个：
CUDA_VISIBLE_DEVICES=1 python run.py --datasets mmlu_ppl ceval_ppl \
--hf-type base \
--hf-path /data/home/shensijia/Model/MERGE/Qwen-sft-ls-v0.1/

**Comment by Leymore** [comment]
请尝试添加 `--debug` 选项，仅进行推理阶段，再贴出错误提示

```
CUDA_VISIBLE_DEVICES=1 python run.py --datasets mmlu_ppl ceval_ppl --hf-type base --hf-path /data/home/shensijia/Model/MERGE/Qwen-sft-ls-v0.1/ --debug -m infer
```

我估计是 huggingface 路径中不能有 '.'

**Reply by 254038lqq** [reply]
你好，这个问题解决了吗？我这边按建议给去掉了路径名中的 ‘.’  但还是有同样的错误

**Reply by kleinzcy** [reply]
请尝试添加 --debug 选项，仅进行推理阶段，再贴出错误提示，有完整报错信息，我们才能更好定位到问题。

**Reply by 254038lqq** [reply]
这是我的执行·代码， python3 run.py --models hf_qwen1_5_7b_chat --datasets mmlu_ppl --debug
在hf_qwen1_5_7b_chat.py  中仅修改了path参数为自己模型的路径，
完整错误有点长，我粘贴前面部分
python3 run.py --models hf_qwen1_5_7b_chat --datasets mmlu_ppl --debug
07/05 09:35:32 - OpenCompass - INFO - Loading mmlu_ppl: configs/datasets/mmlu/mmlu_ppl.py
07/05 09:35:32 - OpenCompass - INFO - Loading hf_qwen1_5_7b_chat: configs/models/qwen/hf_qwen1_5_7b_chat.py
07/05 09:35:32 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
07/05 09:35:32 - OpenCompass - INFO - Current exp folder: outputs/default/20240705_093532
07/05 09:35:32 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
07/05 09:35:32 - OpenCompass - DEBUG - Modules of opencompass's partitioner registry have been automatically imported from opencompass.partitioners
07/05 09:35:32 - OpenCompass - DEBUG - Get class `NumWorkerPartitioner` from "partitioner" registry in "opencompass"
07/05 09:35:32 - OpenCompass - DEBUG - An `NumWorkerPartitioner` instance is built from registry, and its implementation can be found in opencompass.partitioners.num_worker
07/05 09:35:32 - OpenCompass - DEBUG - Key eval.runner.task.judge_cfg not found in config, ignored.
07/05 09:35:32 - OpenCompass - DEBUG - Key eval.runner.task.dump_details not found in config, ignored.
07/05 09:35:32 - OpenCompass - DEBUG - Key eval.given_pred not found in config, ignored.
07/05 09:35:32 - OpenCompass - DEBUG - Additional config: {}
07/05 09:35:32 - OpenCompass - INFO - Partitioned into 1 tasks.
07/05 09:35:32 - OpenCompass - DEBUG - Task 0: [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_biology,qwen1.5-7b-chat-hf/lukaemon_mmlu_college_chemistry,qwen1.5-7b-chat-hf/lukaemon_mmlu_college_computer_science,qwen1.5-7b-chat-hf/lukaemon_mmlu_college_mathematics,qwen1.5-7b-chat-hf/lukaemon_mmlu_college_physics,qwen1.5-7b-chat-hf/lukaemon_mmlu_electrical_engineering,qwen1.5-7b-chat-hf/lukaemon_mmlu_astronomy,qwen1.5-7b-chat-hf/lukaemon_mmlu_anatomy,qwen1.5-7b-chat-hf/lukaemon_mmlu_abstract_algebra,qwen1.5-7b-chat-hf/lukaemon_mmlu_machine_learning,qwen1.5-7b-chat-hf/lukaemon_mmlu_clinical_knowledge,qwen1.5-7b-chat-hf/lukaemon_mmlu_global_facts,qwen1.5-7b-chat-hf/lukaemon_mmlu_management,qwen1.5-7b-chat-hf/lukaemon_mmlu_nutrition,qwen1.5-7b-chat-hf/lukaemon_mmlu_marketing,qwen1.5-7b-chat-hf/lukaemon_mmlu_professional_accounting,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_geography,qwen1.5-7b-chat-hf/lukaemon_mmlu_international_law,qwen1.5-7b-chat-hf/lukaemon_mmlu_moral_scenarios,qwen1.5-7b-chat-hf/lukaemon_mmlu_computer_security,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_microeconomics,qwen1.5-7b-chat-hf/lukaemon_mmlu_professional_law,qwen1.5-7b-chat-hf/lukaemon_mmlu_medical_genetics,qwen1.5-7b-chat-hf/lukaemon_mmlu_professional_psychology,qwen1.5-7b-chat-hf/lukaemon_mmlu_jurisprudence,qwen1.5-7b-chat-hf/lukaemon_mmlu_world_religions,qwen1.5-7b-chat-hf/lukaemon_mmlu_philosophy,qwen1.5-7b-chat-hf/lukaemon_mmlu_virology,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_chemistry,qwen1.5-7b-chat-hf/lukaemon_mmlu_public_relations,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_macroeconomics,qwen1.5-7b-chat-hf/lukaemon_mmlu_human_sexuality,qwen1.5-7b-chat-hf/lukaemon_mmlu_elementary_mathematics,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_physics,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_computer_science,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_european_history,qwen1.5-7b-chat-hf/lukaemon_mmlu_business_ethics,qwen1.5-7b-chat-hf/lukaemon_mmlu_moral_disputes,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_statistics,qwen1.5-7b-chat-hf/lukaemon_mmlu_miscellaneous,qwen1.5-7b-chat-hf/lukaemon_mmlu_formal_logic,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_government_and_politics,qwen1.5-7b-chat-hf/lukaemon_mmlu_prehistory,qwen1.5-7b-chat-hf/lukaemon_mmlu_security_studies,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_biology,qwen1.5-7b-chat-hf/lukaemon_mmlu_logical_fallacies,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_world_history,qwen1.5-7b-chat-hf/lukaemon_mmlu_professional_medicine,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_mathematics,qwen1.5-7b-chat-hf/lukaemon_mmlu_college_medicine,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_us_history,qwen1.5-7b-chat-hf/lukaemon_mmlu_sociology,qwen1.5-7b-chat-hf/lukaemon_mmlu_econometrics,qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_psychology,qwen1.5-7b-chat-hf/lukaemon_mmlu_human_aging,qwen1.5-7b-chat-hf/lukaemon_mmlu_us_foreign_policy,qwen1.5-7b-chat-hf/lukaemon_mmlu_conceptual_physics]
07/05 09:35:32 - OpenCompass - DEBUG - Modules of opencompass's runner registry have been automatically imported from opencompass.runners
07/05 09:35:32 - OpenCompass - DEBUG - Get class `LocalRunner` from "runner" registry in "opencompass"
07/05 09:35:32 - OpenCompass - DEBUG - An `LocalRunner` instance is built from registry, and its implementation can be found in opencompass.runners.local
07/05 09:35:32 - OpenCompass - DEBUG - Modules of opencompass's task registry have been automatically imported from opencompass.tasks
07/05 09:35:32 - OpenCompass - DEBUG - Get class `OpenICLInferTask` from "task" registry in "opencompass"
07/05 09:35:32 - OpenCompass - DEBUG - An `OpenICLInferTask` instance is built from registry, and its implementation can be found in opencompass.tasks.openicl_infer
Error: mkl-service + Intel(R) MKL: MKL_THREADING_LAYER=INTEL is incompatible with libgomp.so.1 library.
        Try to import numpy first or set the threading layer accordingly. Set MKL_SERVICE_FORCE_INTEL to force it.
07/05 09:35:34 - OpenCompass - DEBUG - Get class `NaivePartitioner` from "partitioner" registry in "opencompass"
07/05 09:35:34 - OpenCompass - DEBUG - An `NaivePartitioner` instance is built from registry, and its implementation can be found in opencompass.partitioners.naive
07/05 09:35:34 - OpenCompass - DEBUG - Key eval.runner.task.judge_cfg not found in config, ignored.
07/05 09:35:34 - OpenCompass - DEBUG - Key eval.runner.task.dump_details not found in config, ignored.
07/05 09:35:34 - OpenCompass - DEBUG - Key eval.given_pred not found in config, ignored.
07/05 09:35:34 - OpenCompass - DEBUG - Additional config: {'eval': {'runner': {'task': {}}}}
07/05 09:35:34 - OpenCompass - INFO - Partitioned into 57 tasks.
07/05 09:35:34 - OpenCompass - DEBUG - Task 0: [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_biology]
07/05 09:35:34 - OpenCompass - DEBUG - Task 1: [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_chemistry]
07/05 09:35:34 - OpenCompass - DEBUG - Task 2: [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_computer_science]
07/05 09:35:34 - OpenCompass - DEBUG - Task 3: [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_mathematics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 4: [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_physics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 5: [qwen1.5-7b-chat-hf/lukaemon_mmlu_electrical_engineering]
07/05 09:35:34 - OpenCompass - DEBUG - Task 6: [qwen1.5-7b-chat-hf/lukaemon_mmlu_astronomy]
07/05 09:35:34 - OpenCompass - DEBUG - Task 7: [qwen1.5-7b-chat-hf/lukaemon_mmlu_anatomy]
07/05 09:35:34 - OpenCompass - DEBUG - Task 8: [qwen1.5-7b-chat-hf/lukaemon_mmlu_abstract_algebra]
07/05 09:35:34 - OpenCompass - DEBUG - Task 9: [qwen1.5-7b-chat-hf/lukaemon_mmlu_machine_learning]
07/05 09:35:34 - OpenCompass - DEBUG - Task 10: [qwen1.5-7b-chat-hf/lukaemon_mmlu_clinical_knowledge]
07/05 09:35:34 - OpenCompass - DEBUG - Task 11: [qwen1.5-7b-chat-hf/lukaemon_mmlu_global_facts]
07/05 09:35:34 - OpenCompass - DEBUG - Task 12: [qwen1.5-7b-chat-hf/lukaemon_mmlu_management]
07/05 09:35:34 - OpenCompass - DEBUG - Task 13: [qwen1.5-7b-chat-hf/lukaemon_mmlu_nutrition]
07/05 09:35:34 - OpenCompass - DEBUG - Task 14: [qwen1.5-7b-chat-hf/lukaemon_mmlu_marketing]
07/05 09:35:34 - OpenCompass - DEBUG - Task 15: [qwen1.5-7b-chat-hf/lukaemon_mmlu_professional_accounting]
07/05 09:35:34 - OpenCompass - DEBUG - Task 16: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_geography]
07/05 09:35:34 - OpenCompass - DEBUG - Task 17: [qwen1.5-7b-chat-hf/lukaemon_mmlu_international_law]
07/05 09:35:34 - OpenCompass - DEBUG - Task 18: [qwen1.5-7b-chat-hf/lukaemon_mmlu_moral_scenarios]
07/05 09:35:34 - OpenCompass - DEBUG - Task 19: [qwen1.5-7b-chat-hf/lukaemon_mmlu_computer_security]
07/05 09:35:34 - OpenCompass - DEBUG - Task 20: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_microeconomics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 21: [qwen1.5-7b-chat-hf/lukaemon_mmlu_professional_law]
07/05 09:35:34 - OpenCompass - DEBUG - Task 22: [qwen1.5-7b-chat-hf/lukaemon_mmlu_medical_genetics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 23: [qwen1.5-7b-chat-hf/lukaemon_mmlu_professional_psychology]
07/05 09:35:34 - OpenCompass - DEBUG - Task 24: [qwen1.5-7b-chat-hf/lukaemon_mmlu_jurisprudence]
07/05 09:35:34 - OpenCompass - DEBUG - Task 25: [qwen1.5-7b-chat-hf/lukaemon_mmlu_world_religions]
07/05 09:35:34 - OpenCompass - DEBUG - Task 26: [qwen1.5-7b-chat-hf/lukaemon_mmlu_philosophy]
07/05 09:35:34 - OpenCompass - DEBUG - Task 27: [qwen1.5-7b-chat-hf/lukaemon_mmlu_virology]
07/05 09:35:34 - OpenCompass - DEBUG - Task 28: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_chemistry]
07/05 09:35:34 - OpenCompass - DEBUG - Task 29: [qwen1.5-7b-chat-hf/lukaemon_mmlu_public_relations]
07/05 09:35:34 - OpenCompass - DEBUG - Task 30: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_macroeconomics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 31: [qwen1.5-7b-chat-hf/lukaemon_mmlu_human_sexuality]
07/05 09:35:34 - OpenCompass - DEBUG - Task 32: [qwen1.5-7b-chat-hf/lukaemon_mmlu_elementary_mathematics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 33: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_physics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 34: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_computer_science]
07/05 09:35:34 - OpenCompass - DEBUG - Task 35: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_european_history]
07/05 09:35:34 - OpenCompass - DEBUG - Task 36: [qwen1.5-7b-chat-hf/lukaemon_mmlu_business_ethics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 37: [qwen1.5-7b-chat-hf/lukaemon_mmlu_moral_disputes]
07/05 09:35:34 - OpenCompass - DEBUG - Task 38: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_statistics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 39: [qwen1.5-7b-chat-hf/lukaemon_mmlu_miscellaneous]
07/05 09:35:34 - OpenCompass - DEBUG - Task 40: [qwen1.5-7b-chat-hf/lukaemon_mmlu_formal_logic]
07/05 09:35:34 - OpenCompass - DEBUG - Task 41: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_government_and_politics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 42: [qwen1.5-7b-chat-hf/lukaemon_mmlu_prehistory]
07/05 09:35:34 - OpenCompass - DEBUG - Task 43: [qwen1.5-7b-chat-hf/lukaemon_mmlu_security_studies]
07/05 09:35:34 - OpenCompass - DEBUG - Task 44: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_biology]
07/05 09:35:34 - OpenCompass - DEBUG - Task 45: [qwen1.5-7b-chat-hf/lukaemon_mmlu_logical_fallacies]
07/05 09:35:34 - OpenCompass - DEBUG - Task 46: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_world_history]
07/05 09:35:34 - OpenCompass - DEBUG - Task 47: [qwen1.5-7b-chat-hf/lukaemon_mmlu_professional_medicine]
07/05 09:35:34 - OpenCompass - DEBUG - Task 48: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_mathematics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 49: [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_medicine]
07/05 09:35:34 - OpenCompass - DEBUG - Task 50: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_us_history]
07/05 09:35:34 - OpenCompass - DEBUG - Task 51: [qwen1.5-7b-chat-hf/lukaemon_mmlu_sociology]
07/05 09:35:34 - OpenCompass - DEBUG - Task 52: [qwen1.5-7b-chat-hf/lukaemon_mmlu_econometrics]
07/05 09:35:34 - OpenCompass - DEBUG - Task 53: [qwen1.5-7b-chat-hf/lukaemon_mmlu_high_school_psychology]
07/05 09:35:34 - OpenCompass - DEBUG - Task 54: [qwen1.5-7b-chat-hf/lukaemon_mmlu_human_aging]
07/05 09:35:34 - OpenCompass - DEBUG - Task 55: [qwen1.5-7b-chat-hf/lukaemon_mmlu_us_foreign_policy]
07/05 09:35:34 - OpenCompass - DEBUG - Task 56: [qwen1.5-7b-chat-hf/lukaemon_mmlu_conceptual_physics]
07/05 09:35:34 - OpenCompass - DEBUG - Get class `LocalRunner` from "runner" registry in "opencompass"
07/05 09:35:34 - OpenCompass - DEBUG - An `LocalRunner` instance is built from registry, and its implementation can be found in opencompass.runners.local
07/05 09:35:34 - OpenCompass - DEBUG - Get class `OpenICLEvalTask` from "task" registry in "opencompass"
07/05 09:35:34 - OpenCompass - DEBUG - An `OpenICLEvalTask` instance is built from registry, and its implementation can be found in opencompass.tasks.openicl_eval
07/05 09:35:35 - OpenCompass - DEBUG - Modules of opencompass's load_dataset registry have been automatically imported from opencompass.datasets
07/05 09:35:35 - OpenCompass - DEBUG - Get class `MMLUDataset` from "load_dataset" registry in "opencompass"
07/05 09:35:35 - OpenCompass - DEBUG - An `MMLUDataset` instance is built from registry, and its implementation can be found in opencompass.datasets.mmlu
07/05 09:35:35 - OpenCompass - ERROR - /home/archermind/lqq/opencompass/opencompass/opencompass/tasks/openicl_eval.py - _score - 243 - Task [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_biology]: No predictions found.
07/05 09:35:35 - OpenCompass - DEBUG - Get class `OpenICLEvalTask` from "task" registry in "opencompass"
07/05 09:35:35 - OpenCompass - DEBUG - An `OpenICLEvalTask` instance is built from registry, and its implementation can be found in opencompass.tasks.openicl_eval
07/05 09:35:36 - OpenCompass - DEBUG - Get class `MMLUDataset` from "load_dataset" registry in "opencompass"
07/05 09:35:36 - OpenCompass - DEBUG - An `MMLUDataset` instance is built from registry, and its implementation can be found in opencompass.datasets.mmlu
07/05 09:35:36 - OpenCompass - ERROR - /home/archermind/lqq/opencompass/opencompass/opencompass/tasks/openicl_eval.py - _score - 243 - Task [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_chemistry]: No predictions found.
07/05 09:35:36 - OpenCompass - DEBUG - Get class `OpenICLEvalTask` from "task" registry in "opencompass"
07/05 09:35:36 - OpenCompass - DEBUG - An `OpenICLEvalTask` instance is built from registry, and its implementation can be found in opencompass.tasks.openicl_eval
07/05 09:35:36 - OpenCompass - DEBUG - Get class `MMLUDataset` from "load_dataset" registry in "opencompass"
07/05 09:35:36 - OpenCompass - DEBUG - An `MMLUDataset` instance is built from registry, and its implementation can be found in opencompass.datasets.mmlu
07/05 09:35:36 - OpenCompass - ERROR - /home/archermind/lqq/opencompass/opencompass/opencompass/tasks/openicl_eval.py - _score - 243 - Task [qwen1.5-7b-chat-hf/lukaemon_mmlu_college_computer_science]: No predictions found.
…………
07/05 09:36:13 - OpenCompass - ERROR - /home/archermind/lqq/opencompass/opencompass/opencompass/tasks/openicl_eval.py - _score - 243 - Task [qwen1.5-7b-chat-hf/lukaemon_mmlu_conceptual_physics]: No predictions found.
07/05 09:36:13 - OpenCompass - DEBUG - An `DefaultSummarizer` instance is built from registry, and its implementation can be found in opencompass.summarizers.default
dataset                                            version    metric    mode    qwen1.5-7b-chat-hf
-------------------------------------------------  ---------  --------  ------  --------------------
lukaemon_mmlu_college_biology                      -          -         -       -
lukaemon_mmlu_college_chemistry                    -          -         -       -
lukaemon_mmlu_college_computer_science             -          -         -       -
lukaemon_mmlu_college_mathematics                  -          -         -       -
lukaemon_mmlu_college_physics                      -          -         -       -
lukaemon_mmlu_electrical_engineering               -          -         -       -
lukaemon_mmlu_astronomy                            -          -         -       -
lukaemon_mmlu_anatomy                              -          -         -       -
lukaemon_mmlu_abstract_algebra                     -          -         -       -
lukaemon_mmlu_machine_learning                     -          -         -       -
lukaemon_mmlu_clinical_knowledge                   -          -         -       -
lukaemon_mmlu_global_facts                         -          -         -       -
lukaemon_mmlu_management                           -          -         -       -
lukaemon_mmlu_nutrition                            -          -         -       -
lukaemon_mmlu_marketing                            -          -         -       -
lukaemon_mmlu_professional_accounting              -          -         -       -
lukaemon_mmlu_high_school_geography                -          -         -       -
lukaemon_mmlu_international_law                    -          -         -       -
lukaemon_mmlu_moral_scenarios                      -          -         -       -
lukaemon_mmlu_computer_security                    -          -         -       -
lukaemon_mmlu_high_school_microeconomics           -          -         -       -
lukaemon_mmlu_professional_law                     -          -         -       -
lukaemon_mmlu_medical_genetics                     -          -         -       -
lukaemon_mmlu_professional_psychology              -          -         -       -
lukaemon_mmlu_jurisprudence                        -          -         -       -
lukaemon_mmlu_world_religions                      -          -         -       -
lukaemon_mmlu_philosophy                           -          -         -       -
lukaemon_mmlu_virology                             -          -         -       -
lukaemon_mmlu_high_school_chemistry                -          -         -       -
lukaemon_mmlu_public_relations                     -          -         -       -
lukaemon_mmlu_high_school_macroeconomics           -          -         -       -
lukaemon_mmlu_human_sexuality                      -          -         -       -
lukaemon_mmlu_elementary_mathematics               -          -         -       -
lukaemon_mmlu_high_school_physics                  -          -         -       -
lukaemon_mmlu_high_school_computer_science         -          -         -       -
lukaemon_mmlu_high_school_european_history         -          -         -       -
lukaemon_mmlu_business_ethics                      -          -         -       -
lukaemon_mmlu_moral_disputes                       -          -         -       -
lukaemon_mmlu_high_school_statistics               -          -         -       -
lukaemon_mmlu_miscellaneous                        -          -         -       -
lukaemon_mmlu_formal_logic                         -          -         -       -
lukaemon_mmlu_high_school_government_and_politics  -          -         -       -
lukaemon_mmlu_prehistory                           -          -         -       -
lukaemon_mmlu_security_studies                     -          -         -       -
lukaemon_mmlu_high_school_biology                  -          -         -       -
lukaemon_mmlu_logical_fallacies                    -          -         -       -
lukaemon_mmlu_high_school_world_history            -          -         -       -
lukaemon_mmlu_professional_medicine                -          -         -       -
lukaemon_mmlu_high_school_mathematics              -          -         -       -
lukaemon_mmlu_college_medicine                     -          -         -       -
lukaemon_mmlu_high_school_us_history               -          -         -       -
lukaemon_mmlu_sociology                            -          -         -       -
lukaemon_mmlu_econometrics                         -          -         -       -
lukaemon_mmlu_high_school_psychology               -          -         -       -
lukaemon_mmlu_human_aging                          -          -         -       -
lukaemon_mmlu_us_foreign_policy                    -          -         -       -
lukaemon_mmlu_conceptual_physics                   -          -         -       -
07/05 09:36:13 - OpenCompass - INFO - write summary to /home/archermind/lqq/opencompass/opencompass/outputs/default/20240705_093532/summary/summary_20240705_093532.txt
07/05 09:36:13 - OpenCompass - INFO - write csv to /home/archermind/lqq/opencompass/opencompass/outputs/default/20240705_093532/summary/summary_20240705_093532.csv

**Reply by kleinzcy** [reply]
Error: mkl-service + Intel(R) MKL: MKL_THREADING_LAYER=INTEL is incompatible with libgomp.so.1 library.
Try to import numpy first or set the threading layer accordingly. Set MKL_SERVICE_FORCE_INTEL to force it.

推理阶段有这个bug。你检查一下你最近的几次（除了--debug这次）output输出文件，outputs/default/日期/prediction 目录是否存在输出内容，如果内容是空，则很可能是上述error导致推理失败。

此外，outputs/default/日期/infer目录包含推理过程中的输出信息，你可以根据这个目录里面的信息debug

**Reply by 254038lqq** [reply]
![20240705095316](https://github.com/open-compass/opencompass/assets/76122750/12cfa5c5-8054-4fda-b654-ce637544d26f)
outputs/default/日期/   目录中只有configs 与 summary 两个文件夹，并没有infer文件夹，
如果我将  python3 run.py --models hf_qwen1_5_7b_chat --datasets mmlu_ppl --debug   后面加上 -m infer的话，确实会有07/05 09:47:53 - OpenCompass - DEBUG - An `OpenICLInferTask` instance is built from registry, and its implementation can be found in opencompass.tasks.openicl_infer
Error: mkl-service + Intel(R) MKL: MKL_THREADING_LAYER=INTEL is incompatible with libgomp.so.1 library.
        Try to import numpy first or set the threading layer accordingly. Set MKL_SERVICE_FORCE_INTEL to force it.这个error提示，

**Reply by kleinzcy** [reply]
你检查一下你最近的几次（**除了--debug这次**）output输出文件， --debug 参数会跳过实际推理过程

**Reply by 254038lqq** [reply]
![132456](https://github.com/open-compass/opencompass/assets/76122750/96de3cf7-cf6d-4dc5-98bc-58f0cf5c252a)
我将命令中的debug参数去掉，python3 run.py --models hf_qwen1_5_7b_chat --datasets mmlu_ppl跑完之后目录也是之前的那样

**Reply by Mor-Li** [reply]
可以先尝试解决下面的bug，也许这个[链接](https://stackoverflow.com/questions/66932956/python-mkl-threading-layer-intel-is-incompatible-with-libgomp-so-1-library)会有帮助？
`Error: mkl-service + Intel(R) MKL: MKL_THREADING_LAYER=INTEL is incompatible with libgomp.so.1 library.
Try to import numpy first or set the threading layer accordingly. Set MKL_SERVICE_FORCE_INTEL to force it.`

**Reply by kleinzcy** [reply]
那应该就是因为上述error，导致infer没有启动，infer文件夹也就没有被创建。先解决上述error，再看看能否正常运行。

**Reply by 254038lqq** [reply]
好的，非常感谢，我先看一下上述的问题
